src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
Remaining calc's are:

			// if cycling fan, send coil part-load fraction to on/off fan via HVACDataGlobals
			if ( FanOpMode == CycFanCycCoil ) OnOffFanPartLoadFraction = PLF;
			DXCoil( DXCoilNum ).ElecHeatingPower = TotCap * EIR * DXCoil( DXCoilNum ).HeatingCoilRuntimeFraction * InputPowerMultiplier;

			// Calculate crankcase heater power using the runtime fraction for this DX heating coil only if there is no companion DX coil.
			// Else use the largest runtime fraction of this DX heating coil and the companion DX cooling coil.

			if ( DXCoil( DXCoilNum ).CompanionUpstreamDXCoil == 0 ) {
				DXCoil( DXCoilNum ).CrankcaseHeaterPower = CrankcaseHeatingPower * ( 1.0 - DXCoil( DXCoilNum ).HeatingCoilRuntimeFraction );
			} else {
				DXCoil( DXCoilNum ).CrankcaseHeaterPower = CrankcaseHeatingPower * ( 1.0 - max( DXCoil( DXCoilNum ).HeatingCoilRuntimeFraction, DXCoil( DXCoil( DXCoilNum ).CompanionUpstreamDXCoil ).CoolingCoilRuntimeFraction ) );
			}

			AirMassFlow = DXCoil( DXCoilNum ).InletAirMassFlowRate;
			DXCoil( DXCoilNum ).TotalHeatingEnergyRate = AirMassFlow * ( OutletAirEnthalpy - InletAirEnthalpy );
			// Adjust defrost power to correct for DOE-2 bug where defrost power is constant regardless of compressor runtime fraction
			// Defrosts happen based on compressor run time (frost buildup on outdoor coil), not total elapsed time.
			DXCoil( DXCoilNum ).DefrostPower *= DXCoil( DXCoilNum ).HeatingCoilRuntimeFraction;

			DXCoil( DXCoilNum ).OutletAirTemp = OutletAirTemp;
			DXCoil( DXCoilNum ).OutletAirHumRat = OutletAirHumRat;
			DXCoil( DXCoilNum ).OutletAirEnthalpy = OutletAirEnthalpy;
			DXCoil( DXCoilNum ).CompressorPartLoadRatio = PartLoadRatio;